<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Advice</class>
 <widget class="QMainWindow" name="Advice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>400</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Advice</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>508</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="Bt_Obese">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>51</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;MV Boli&quot;;
color: rgb(0, 0, 0);
background-color: rgb(255, 0, 0);</string>
       </property>
       <property name="text">
        <string>อ้วน( Obese ) | BMI  &gt;25</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_Overweight">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;MV Boli&quot;;
background-color: rgb(255, 237, 34);
color: rgb(0, 0, 0);</string>
       </property>
       <property name="text">
        <string>น้ำหนักเกิน(Overweight ) | BMI : 23-24.9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_Normal">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;MV Boli&quot;;
background-color: rgb(170, 255, 0);</string>
       </property>
       <property name="text">
        <string>น้ำหนักปกติ(Normal) | BMI : 18.5-22.9</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="Bt_Underweight">
       <property name="styleSheet">
        <string notr="true">font: 15pt &quot;MV Boli&quot;;</string>
       </property>
       <property name="text">
        <string>น้ำหนักน้อย(Underweight) | BMI : &lt;18.5</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>781</width>
      <height>51</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="minimumSize">
        <size>
         <width>361</width>
         <height>41</height>
        </size>
       </property>
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:14pt; font-weight:600;&quot;&gt;รายการคำแนะนำ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="HomeButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;home&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 24pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(85, 170, 127);</string>
    </property>
    <property name="text">
     <string>Home</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_YourBMI">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>80</y>
      <width>291</width>
      <height>61</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;YourBMI&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);
font: 63 36pt &quot;Yu Gothic UI Semibold&quot;;</string>
    </property>
    <property name="text">
     <string>Your BMI :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_ShowBMI">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>151</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 36pt &quot;Yu Gothic UI Semibold&quot;;
color: rgb(0, 170, 255);</string>
    </property>
    <property name="text">
     <string>:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
